src/structures/interfaces/TextBasedChannel.js

```diff
@@ -110,30 +110,23 @@ class TextBasedChannel {
   async send(content, options) {
     const User = require('../User');
     const GuildMember = require('../GuildMember');
+
     if (this instanceof User || this instanceof GuildMember) {
       return this.createDM().then(dm => dm.send(content, options));
     }
 
-    const apiMessage = APIMessage.create(this, content, options);
-    const data = apiMessage.resolveData();
-    if (data.content instanceof Array) {
-      const messages = [];
-      for (let i = 0; i < data.content.length; i++) {
-        let opt;
-        if (i === data.content.length - 1) {
-          opt = { tts: data.tts, embed: data.embed, files: apiMessage.options.files };
-        } else {
-          opt = { tts: data.tts };
-        }
+    let apiMessage;
 
-        // eslint-disable-next-line no-await-in-loop
-        const message = await this.send(data.content[i], opt);
-        messages.push(message);
+    if (content instanceof apiMessage) {
```

Comment:
This should be `APIMessage` as `apiMessage` is the variable intended for an instance later.